librz/include/rz_util/rz_base64.h

```diff
@@ -8,8 +8,8 @@ extern "C" {
 #endif
 
 RZ_API size_t rz_base64_encode(char *bout, const ut8 *bin, size_t sz);
-RZ_API int rz_base64_decode(ut8 *bout, const char *bin, int len);
-RZ_API ut8 *rz_base64_decode_dyn(const char *in, int len);
+RZ_API st64 rz_base64_decode(ut8 *bout, const char *bin, st64 len);
+RZ_API ut8 *rz_base64_decode_dyn(const char *in, st64 len);
```

Comment:
While you are on it, could you please also add ownership marks? Like described in DEVELOPERS.md